packages/documentation/docs/controls/forms/_forms-behavior_styleguide.md

```diff
@@ -0,0 +1,42 @@
+Forms behavior refers to the way in which user input is handled and validated within a form. It plays a crucial role in providing a seamless and user-friendly experience for form interactions.
+
+## Validation Feedback
+When data is validated, a validation result is provided to guide users in correcting invalid data, convey a sense of security, warn about critical inputs, or inform about special features of an input.
+
+We support 4 types of validation feedback.
+
+##### Valid
+- native HTML status
+- data meets all validity criteria
+##### Invalid
+- native HTML status
+- the data violates at least one validity criterion
+- invalid inputs must be corrected before they can be used
+##### Warning
+- Custom status
+- The data is valid in general but violates a specific security or safety related rule
+- Example: the entered rotation speed is beyond a safety threshold and can cause higher wear
+##### Info
+- Custom status
+- The data is valid but fulfills a certain criterion
+- Example: highlight all inputs that has been changed compared to a saved configuration
+- For more information see #Validation
+
+## Submit Button Strategies
```

Comment:
Forms are also valid to submit on change, without any dedicated submit action

Reply:
Remove Button - add 3 Strategy: on blur

Combination 1 + 3